README.md

```diff
@@ -1,26 +1,40 @@
 # Pennsieve App
```

Comment:
I think these are changes that were committed directly to `main` 🤷‍♂️ let me know if you want me to add a commit that reverts them for `dev` but that would put `dev` in a weird state compared to `main`...

Reply:
Yeah, thats my bad, I didn't update dev with the README changes. These are fine to piggyback on this PR.